astropy/visualization/wcsaxes/utils.py

```diff
@@ -14,7 +14,7 @@
     "transform_contour_set_inplace",
 ]
 
-MATPLOTLIB_LT_3_8 = not minversion(matplotlib, "3.8.dev")
+MATPLOTLIB_LT_3_8 = not minversion(mpl, "3.8.dev")
```

Comment:
For a case like this, we don't actually have to import `matplotlib` at all, just do `minversion("matplotlib", "3.8.dev")`
",